cocotb/share/lib/verilator/verilator.cpp

```diff
@@ -111,11 +111,15 @@ int main(int argc, char** argv) {
 #endif
         // cocotb controls the clock inputs using cbAfterDelay so
         // skip ahead to the next registered callback
-        vluint64_t next_time = VerilatedVpi::cbNextDeadline();
+        const vluint64_t NO_TOP_EVENTS_PENDING = static_cast<vluint64_t>(~0ULL);
```

Comment:
Technically this could be `constexpr`, but who cares. The compiler is going to const prop it regardless.